helm/templates/daemonset.yaml

```diff
@@ -67,7 +67,9 @@ spec:
             - name: efs-state-dir
               mountPath: /var/run/efs
             - name: efs-utils-config
-              mountPath: /etc/amazon/efs
+              mountPath: /var/amazon/efs
```

Comment:
I am not 100% sure but I think changes to the helm chart have to be made at the same time as or after the release containing these code changes, i.e. 1.1.0/1.0.1. Otherwise the current helmchart 0.3.0 with driver version 1.0.0 won't work anymore.

LEaving this comment so i can check and be sure...

Reply:
This change was needed to get CI to pass. I don't understand all the packaging implications, but if we back it out we should expect to CI fail. It was this push that got CI to pass https://github.com/kubernetes-sigs/aws-efs-csi-driver/compare/dcee1e279142b1cdcdc452ab2d688f7077788c19..7fc9f513ab23e9a94470f0311575d9f8011e3195

Reply:
O yeah, makes sense, thanks let's keep it

Reply:
You are right, with the new install instruction only after the index gets updated would chart be consumable